src/features/tools/components/tool-description.jsx

```
@@ -0,0 +1,61 @@
+    >
+      <div>
+        <h2>{label}</h2>
+        <small>
+          <ReactMarkdown
+            remarkPlugins={[remarkGfm]}
+            rehypePlugins={[rehypeRaw]}
+            components={{
+              a: ({ href, children, ...props }) => {
+                if (isElectron())
+                  return (
+                    <a {...props} onClick={() => openExternal(href)}>
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)